banzai/tests/e2e-k8s.yaml

```diff
@@ -76,6 +76,7 @@ spec:
             - node_health_check
         initialDelaySeconds: 5
         periodSeconds: 1
+        timeoutSeconds: 10
```

Comment:
Update the timeoutSeconds on the readiness probes because kubernetes would sometimes kill the container if it didn't respond within 1 second, so give it a bit more slack.

Reply:
should we be concerned about this? Is this a problem caused by the new map?

Reply:
I've had to update this up to 15 seconds on banzai-floyds. Rabbitmq is taking a lot longer to wake up than it used to.